src/Twig/StorageExtension.php

```diff
@@ -24,8 +26,14 @@ public function getFilters(): array
         ];
     }
 
-    public function formatUrl(string $path): string
+    public function formatUrl(string $path, AssetsHelper $assetsHelper): string
     {
-        return $this->awsS3PublicUrlResolver->resolve($path);
+        // dump( preg_match(self::AUTHORIZED_SCHEMA_REGEX, $path) );
```

Comment:
Remove this dump

Reply:
Je ne l'ai plus dans mon code  tu es sur d'avoir la bonne version ?

Reply:
Indeed, hence the `Outdated` tag next to the file name. There is a `resolve conversation` button for that 😄